frontend/src/views/BookDetail.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <div>
+    <div>
+      <table class="table table-hover">
+        <tbody v-if="book">
+          <tr>
+            <td>Id</td>
+            <td>{{ book.id }}</td>
+          </tr>
+          <tr>
+            <td>Article Count</td>
+            <td>{{ book.article_count }}</td>
+          </tr>
+          <tr>
+            <td>Media Count</td>
+            <td>{{ book.media_count }}</td>
+          </tr>
+          <tr>
+            <td>Counter</td>
+            <td>
+              <pre>{{ book.counter }}</pre>
+            </td>
+          </tr>
+          <tr>
+            <td>Period</td>
+            <td>{{ book.period }}</td>
+          </tr>
+          <tr>
+            <td>Size</td>
+            <td>{{ $filters.bytesToHumanReadable(book.size) }}</td>
+          </tr>
+          <tr>
+            <td>Url</td>
+            <td>
+              <a href="{{ book.url }}">{{ book.url }}</a>
+            </td>
+          </tr>
+          <tr>
+            <td>Metadata</td>
+            <td>
+              <table>
+                <tbody>
+                  <tr
+                    v-for="(value, key) in book.metadata"
+                    :key="value"
+                  >
+                    <td>
+                      {{ key }}
+                    </td>
+                    <td v-if="key === 'Illustration_48x48'">
```

Comment:
Should check for the kind ; not this very specific name as ZIM can have multiple illustratiobs

Reply:
Hi @rgaudin  I am using the `startsWith('Illustration_')`

because in order to add "kind" data to the response I would have to change a lot of structure

and we use `startswith()` for the same work in Python

Reply:
OK that's good enough for now